src/Effects.js

```diff
@@ -1541,6 +1541,40 @@ module.exports = class Effects {
       }
     };
 
+    this.growing_stars = {
+      stars: [],
+      colorIndex: 0,
+      starSpacing: 30,
+      numStars: 9,
+
+      init: function () {
+        p5.angleMode(p5.RADIANS);
+        this.colorIndex = 0;
```

Comment:
Do we need to set these to 0 (and []) in init if they are initialized as such at the top? Or does this have to do with resetting?

Reply:
Yeah we need to reset these in `init` so that if you run/reset/run it works as expected